packages/adapter/addon/rest.js

```diff
@@ -1366,7 +1366,7 @@ function headersToObject(headers) {
  * @returns {Object}
  */
 export function fetchOptions(options, adapter) {
-  options.credentials = 'same-origin';
+  options.credentials = options.credentials || 'same-origin';
```

Comment:
👋 Possible we can add a test for this?  Otherwise, this looks great!

Reply:
I will modify the source so that only the existing tests can be passed and used.

Reply:
I didn't pass all of your tests 😥
How is there no way?

Reply:
@kookyungmin Mind merging in master?  Tests should pass now.

Reply:
Thank you. Let me check.